<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsMeasSetup</class>
 <widget class="QDialog" name="clsMeasSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置测试参数</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myIcon.qrc">
    <normaloff>:/Icon/Icons/wk.ico</normaloff>:/Icon/Icons/wk.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	
	font: 12pt &quot;楷体&quot;;
	background-color: rgb(148, 148, 248);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>测量项目</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>项目1：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnItem1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>项目2：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnItem2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnEqucct">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>串联</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>驱动信号</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>频率：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFreq">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>100k</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>电平：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLevel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>1.00V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>偏置设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>大小：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBias">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>0.00V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnBiasType">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>电流</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnSwitch">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>关</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
     </property>
     <property name="title">
      <string>档位速度</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnSpeed">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>快速</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnRange">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>33</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myIcon.qrc"/>
 </resources>
 <connections/>
</ui>
